multinode_integration_tests/tests/blockchain_interaction_test.rs

```diff
@@ -1,29 +1,40 @@
 // Copyright (c) 2019, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
 
-use std::time::Duration;
+use std::ops::Add;
+use std::path::PathBuf;
+use std::time::{Duration, SystemTime};
 
 use log::Level;
 use regex::escape;
 use serde_derive::Serialize;
 
-use masq_lib::messages::{ScanType, ToMessageBody, UiScanRequest};
+use masq_lib::messages::{FromMessageBody, ScanType, ToMessageBody, UiScanRequest, UiScanResponse};
+use masq_lib::test_utils::utils::is_running_under_github_actions;
 use masq_lib::utils::find_free_port;
 use multinode_integration_tests_lib::masq_node::MASQNode;
 use multinode_integration_tests_lib::masq_node::MASQNodeUtils;
 use multinode_integration_tests_lib::masq_node_cluster::MASQNodeCluster;
 use multinode_integration_tests_lib::masq_real_node::{
-    ConsumingWalletInfo, NodeStartupConfigBuilder,
+    ConsumingWalletInfo, MASQRealNode, NodeStartupConfigBuilder,
 };
 use multinode_integration_tests_lib::mock_blockchain_client_server::MBCSBuilder;
-use multinode_integration_tests_lib::utils::{config_dao, database_conn, receivable_dao};
+use multinode_integration_tests_lib::utils::{
+    config_dao, open_all_file_permissions, receivable_dao, UrlHolder,
+};
+use node_lib::sub_lib::wallet::Wallet;
 
 #[test]
-#[ignore]
 fn debtors_are_credited_once_but_not_twice() {
+    if is_running_under_github_actions() {
+        eprintln!("This test doesn't pass under GitHub Actions; don't know why");
+        return;
+    }
     let mbcs_port = find_free_port();
     let ui_port = find_free_port();
+    let mut cluster = MASQNodeCluster::start().unwrap();
     // Create and initialize mock blockchain client: prepare a receivable at block 2000
-    let _blockchain_client_server = MBCSBuilder::new(mbcs_port)
+    eprintln!("Setting up mock blockchain client");
```

Comment:
You left eprintlns behind you all over this test's body. Of course, excluding the one in the if statement at the beginning.

Reply:
Good catch!